source/extensions/omni.isaac.lab_tasks/test/test_environments.py

```diff
@@ -49,13 +49,13 @@ def test_multiple_instances_gpu(self):
         """Run all environments with multiple instances and check environments return valid signals."""
         # common parameters
         num_envs = 32
-        use_gpu = True
+        device = "cuda"
```

Comment:
Should we default everywhere to "cuda" in sim context cfg? It seems that in the test you are switching to "cuda" everywhere?